examples/meta/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@
 # execute_process(COMMAND sh ${CMAKE_CURRENT_SOURCE_DIR}/generator/types/get_type_list.sh > ${CMAKE_CURRENT_SOURCE_DIR}/generator/types/typelist)
 
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/parser_files)
+include_directories(${CEREAL_INCLUDE_DIRS})
```

Comment:
ok i understand this... buuuuut and here comes the *pain* part: in the top level CMakeLists.txt you do this:
https://github.com/OXPHOS/shogun/blob/86425a27566deec4f8eb065af5526f23571a031c/CMakeLists.txt#L595

a.k.a. what we did for eigen. if you end up doing the internal dependency to have this working (above) you would need to ship cereal with shogun in the header directory... which is quite a problem.... only way to solve this if we could add all the cereal related implementations/definitions (declaration can go into headers as that can be easily solved) into .cpp